### PR TITLE
skill section enhanced

### LINKED ISSUE #476: Sharpen your skill enhancement
## Proposal
the  current section is not aligned properly  or we can convert it into cards  which will look better or we can align it center 

(A clear and concise description of what the proposal is.)
Algo Phantoms - Google Chrome 12-05-2021 22_39_37

### PR DESCRIPTION
**Pull request by Ankita297**
# Description

skill section enhanced
Fixes #476

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
    - After :
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Algo Phantoms - Google Chrome 20-05-2021 21_42_07
Algo Phantoms - Google Chrome 20-05-2021 21_43_09

Algo Phantoms - Google Chrome 20-05-2021 21_39_31

https://user-images.githubusercontent.com/67833077/119014558-b7467800-b9b5-11eb-82c2-63bb98e73bc2.mp4

### CODE REVIEW

**Comment by Ankita297** [comment]
@harikesh409 @ravikr126 Sir Please review my PR .

**Review by ravikr126 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -196,25 +196,59 @@ <h1 class="text-center">About Us</h1>
...
+                <div class="col-lg-4 col-md-4 col-sm-12">
+                    <div class="card skill-card">
+                        <h5><i class="fab fa-swift"></i>&nbsp; <a href="https://www.youtube.com/watch?v=Ulp1Kimblg0" target="_blank">Swift</a></h5>
+                    </div>
+                </div>
+            </div>
+        </div>
     </div>
     </section>
     <span class="span"></span>
```
Comment (harikesh409) [inline index.html:254]: Remove `<span class="span"></span>` this which is not necessary.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
There are few dead styles present in `style.css` like 
- .skills-section table
- .skills-section th:hover
- .skills-section th,td
- .skills-section tr:nth-child(odd) th
- .skills-section tr:nth-child(odd) th:hover
- .skills-section tr:nth-child(even) th:hover
Remove these properties.

**Comment by Ankita297** [comment]
@harikesh409 Sir Please see now .

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**